### ISSUE TITLE
error on helm diff command

### ISSUE THREAD

**Issue by Dmitry1987** [databus23/helm-diff#107]
Hi,
I installed as shown in the readme, the 'diff' plugin was added to helm and shows 'usage', so I guess it was installed properly, but when I try the diff, using this command:
```
helm diff upgrade somerelease charts/somechart -f values/somevalues.yaml
```
it says
```
portforward.go:316] error copying from local connection to remote stream: read tcp4 127.0.0.1:37929->127.0.0.1:40070: read: connection reset by peer
```
some local connection issue.  But helm works and everything else too (kubectl etc')

**Comment by andytom** [comment]
This has been reported upstream https://github.com/helm/helm/issues/3480 (and also in https://github.com/roboll/helmfile/issues/396).

**Comment by willejs** [comment]
That was fixed in 2.13. If you run 2.13 tiller and helm client, this does not happen. The helm dep in this repo needs to be upgraded to 2.13 too.

**Comment by Dmitry1987** [comment]
Thanks @willejs
